simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -240,7 +240,7 @@ public T rdfTermFactory(RDF rdfTermFactory) {
     public T contentType(RDFSyntax rdfSyntax) throws IllegalArgumentException {
         AbstractRDFParser<T> c = clone();
```

Comment:
Is it actually necessary to clone here? The typical builder pattern is much simpler with its reliance on always having "return this" as its last line, and using mutation before the "build()" terminal method to reduce this horrendous complexity related to cloning longterm mutable objects.